src/lib.ts

```diff
@@ -119,10 +120,7 @@ export default async function isMergable(opts: Options): Promise<Result> {
 		pendingReviewRequests.push({ name: reviewRequest.login })
 	}
 
-	let latestStatuses: Map<
-		string,
-		Octokit.ReposListStatusesForRefResponseItem
```

Comment:
Does this type just not exist anymore?

Reply:
They moved to a plugin model and the type is no longer exported it.  Actually I found `@octokit/types`, this looks like the correct option. Let me update to use that.

Reply:
So I updated to use this type library, however there is some odd typescript thing happening. It is erroring on properties that typescript has the object is declared as having. Unfortunately, I don't know typescript well enough to debug it further. I attempted to add circle so it would be easier to show the error but I didn't have enough permissions.